src/main/java/uk/gov/digital/ho/hocs/search/client/elasticsearchclient/ElasticSearchClient.java

```diff
@@ -16,16 +15,9 @@
 import org.elasticsearch.search.builder.SearchSourceBuilder;
 import uk.gov.digital.ho.hocs.search.domain.exceptions.ApplicationExceptions;
 import uk.gov.digital.ho.hocs.search.domain.model.CaseData;
-
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
We shouldn't be bringing imports for entire packages? There is an IntelliJ setting that can be changed to stop this.